pkg/ddc/jindofsx/transform.go

```diff
@@ -1133,3 +1134,15 @@ func (e *JindoFSxEngine) getMediumTypeFromVolumeSource(defaultMediumType string,
 
 	return mediumType
 }
+
+func (e *JindoFSxEngine) checkVersionSupportAkFile(runtime *datav1alpha1.JindoRuntime, smartdataTag string, fuseTag string) bool {
```

Comment:
rename  checkVersionSupportAkFile to checkIfSupportSecretMount?

Reply:
DONE